sn_client/src/folders.rs

```diff
@@ -196,14 +205,55 @@ impl FoldersApi {
         Ok(())
     }
 
+    /// Sync multiple Folders with the network.
+    pub async fn sync_multiple(
```

Comment:
Not sure this is correct here, all methods exposed by this struct shall be related to itself, a single Folder. If there is some upper layer holding a bunch of Folders then that's where this method/fn belongs to, not here. We do this in the acc-packet as an example, which is where we keep a set of folders. 

This method needs to be removed from this struct and moved yup to the caller layer which is the only holding all the folders.